internal/keycloakb/eventsdbmodule.go

```diff
@@ -169,6 +191,110 @@ func (cm *eventsDBModule) GetTotalConnectionsCount(_ context.Context, realmName
 	return res, err
 }
 
+// GetTotalConnectionsHoursCount gets the number of connections for the given realm for the last 24 hours, hour by hour
+func (cm *eventsDBModule) GetTotalConnectionsHoursCount(_ context.Context, realmName string) ([][]int64, error) {
+
+	var err error
+	var nbHours = 24
+	var nbConnections = int64(0)
+	var res = make([][]int64, nbHours)
+	for i := 0; i < nbHours; i++ {
```

Comment:
It would be good to avoid to perform a query for each Hours.
You could use something like the following : https://popsql.com/learn-sql/mysql/how-to-group-by-time-in-mysql/
It would directly returns all the values, thus it would be better for performance.

This can be used for all our queries per hours, per days, per months...

Reply:
DONE, the code and the queries were updated